itests/standalone/docker/src/test/java/org/wildfly/camel/test/stomp/StompIntegrationTest.java

```diff
@@ -33,55 +33,61 @@
 import org.apache.camel.builder.RouteBuilder;
 import org.apache.camel.component.mock.MockEndpoint;
 import org.apache.camel.impl.DefaultCamelContext;
+import org.arquillian.cube.CubeController;
+import org.arquillian.cube.docker.impl.requirement.RequiresDocker;
 import org.fusesource.stomp.client.BlockingConnection;
 import org.fusesource.stomp.client.Stomp;
 import org.fusesource.stomp.codec.StompFrame;
 import org.jboss.arquillian.container.test.api.Deployment;
 import org.jboss.arquillian.junit.Arquillian;
 import org.jboss.arquillian.test.api.ArquillianResource;
-import org.jboss.as.arquillian.api.ServerSetup;
-import org.jboss.as.arquillian.api.ServerSetupTask;
-import org.jboss.as.arquillian.container.ManagementClient;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.wildfly.camel.test.common.utils.JMSUtils;
+import org.wildfly.camel.test.common.utils.TestUtils;
 import org.wildfly.extension.camel.CamelAware;
 
 /**
  * @author <a href="https://github.com/ppalaga">Peter Palaga</a>
  */
 @CamelAware
+@RequiresDocker
 @RunWith(Arquillian.class)
```

Comment:
Ah, caught out with the same mistake I made recently....

For the `RequiresDocker` annotation to work, it must be used in conjunction with the `ArquillianConditionalRunner` instead of the normal Arquillian test runner.

Reply:
Thanks, fixed in f56c111ea3840f655361f52c0c4b1dc1676a1b4f